contrib/action_recognition/README.md

```diff
@@ -1,28 +1,41 @@
 # Action Recognition
 
-This is a place holder. Content will follow soon.
+This folder contains projects relevant to video-based action recognition.
 
-![](./media/action_recognition.gif)
+![](./media/action_recognition.gif "Example of action recognition")
 
 *Example of action recognition*
 
 ## Overview
 
 | Folders |  Description |
 | -------- |  ----------- |
-| [i3d](i3d)  | Scripts for fine-tuning a pre-trained Two-Stream Inflated 3D ConvNet (I3D) model on the HMDB-51 dataset
+| [i3d](i3d)  | Scripts for fine-tuning a pre-trained Two-Stream Inflated 3D ConvNet (I3D) model on the [HMDB-51 dataset](http://serre-lab.clps.brown.edu/resource/hmdb-a-large-human-motion-database/)
+| [r2p1d](r2p1d)  | [R(2+1)D model](https://arxiv.org/abs/1711.11248) scripts and notebooks for fine-tuning a pre-trained on the HMDB-51 dataset and webcam demo
 | [video_annotation](video_annotation)  | Instructions and helper functions to annotate the start and end position of actions in video footage|
 
 ## Functionality
 
+### Models
+
 In [i3d](i3d) we show how to fine-tune a Two-Stream Inflated 3D ConvNet (I3D) model. This model was introduced in \[[1](https://arxiv.org/pdf/1705.07750.pdf)\] and achieved state-of-the-art in action classification on the HMDB-51 and UCF-101 datasets. The paper demonstrated the effectiveness of pre-training action recognition models on large datasets - in this case the Kinetics Human Action Video dataset consisting of 306k examples and 400 classes. We provide code for replicating the results of this paper on HMDB-51. We use models pre-trained on Kinetics from [https://github.com/piergiaj/pytorch-i3d](https://github.com/piergiaj/pytorch-i3d). Evaluating the model on the test set of the HMDB-51 dataset (split 1) using [i3d/test.py](i3d/test.py) should yield the following results:
 
-| Model | Paper top 1 accuracy (average over 3 splits) | Our models top 1 accuracy (split 1 only) |
+In [r2p1d](r2p1d) we demonstrate fine-tuning R(2+1)D model [pretrained on 65 million videos](https://arxiv.org/abs/1905.00561).
```

Comment:
Move r2p1d up, since that's the one we recommend people to use. could also mention that it runs in (near) real-time. If possible, highlight what i3d and r2p1d have in common, and what we only wrote for r2p1d